papas/detectors/CMS.py

```diff
@@ -133,7 +137,17 @@ def __init__(self, magnitude):
         volume = VolumeCylinder('field', 2.9, 3.6)
         mat = material.void
         super(Field, self).__init__('tracker', volume,  mat)
-        
+
+class BeamPipe(DetectorElement):
+
+    def __init__(self):
+        #Material Seamless AISI 316 LN, External diameter 53 mm, Wall thickness 1.5 mm (hors cms) X0 1.72 cm
+        #in CMS, radius 25 mm (?), tchikness 8mm, X0 35.28 cm : berylluim
+        factor = 1.0
```

Comment:
remove factor.
Btw what is this for? I hope it's not to modify the amount of material, cause it's going to change also the inner radius of the beam pipe. 
What is the difference between zmin and zmax? 1.98 vs 1.9785